taxonomies/packaging_recycling.result.txt

```diff
@@ -1,13 +1,14 @@
 
 
 en:discard, to discard, throw away, to throw away, non-recyclable
+de:wegwerfen, nicht recycelbar, zum Wegwerfen
 fr:jeter, à jeter, non-recyclable
-pt:descartar, a descartar, para descartar, não-reciclável, não recicláel
 
 en:recycle, to recycle, to be recycled, recyclable
+de:recyceln, zum Recyceln, recycelbar
 fr:recycler, à recycler, recyclable
-pt:reciclável, a reciclar, para reciclar
```

Comment:
why is pt disappearing ?

Reply:
because they were added to the results.txt file instead of the source file: https://github.com/openfoodfacts/openfoodfacts-server/commit/7b620fb033da59ec701cc8f0572dac57202f9e55#diff-ee7816924647a1758e27c2b5f8bb182e68e468e2f2a87bb10fc9f00213e006a8